autodock_core/scripts/autodock_core/autodock_server.py

```diff
@@ -336,6 +344,19 @@ def rotate_with_odom(self, rotate: float) -> bool:
             rospy.sleep(self.sleep_period)
         exit(0)
 
+    def set_aruco_detections(self, detection_state) -> bool:
+        """
+        Set aruco detections to True or False
+        :return : success
+        """
+        try:
```

Comment:
As you have recommended, I think having a debug mode (default on) is helpful for people to test this package out during the integration process. we can define a ros param `debug` and set it in the launch file. if debug is off, this `set_aruco_detections` function will just return true without calling the service call.